pkg/asset/metadata/metadata.go

```diff
@@ -0,0 +1,80 @@
+package metadata
+
+import (
+	"encoding/json"
+	"fmt"
+
+	// "github.com/hashicorp/terraform/terraform"
```

Comment:
Are you planning on using this for something?  I'd rather not land new commented-out code unless we're pretty sure we're going to consume it in the next week or so.

Reply:
removed.